app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/Nullable.tsx

```diff
@@ -37,6 +37,11 @@ const NullableBase = ({ nullable, onNullable: onChange }: INullableBaseProps) =>
       }}
       disabled={typeof onChange !== 'function'}
       data-testid="schema-field-nullable-checkbox"
+      title={
```

Comment:
I'm not sure that the tooltip should change based on the checkbox status. Can you check with UX?

Reply:
Thanks for comment. Yes you are correct, dynamically changing tooltip based on checkbox state is not suggested.
Now, changed to give  a single tooltip which conveys context and purpose. Changed the description as well, PTAL.